test/cql-pytest/test_guardrail_compact_storage.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from util import config_value_context, new_test_keyspace, new_test_table
+from cassandra_tests.porting import *
+
+# Tests for the enable_create_table_with_compact_storage guardrail.
+# Because this feature does not exist in Cassandra , *all* tests in this file are
+# Scylla-only. Let's mark them all scylla_only with an autouse fixture:
+@pytest.fixture(scope="function", autouse=True)
+def all_tests_are_scylla_only(scylla_only):
+    pass
+
+def test_create_table_with_compact_storage_default_config(cql, this_dc):
+    ks_defs = f"WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 1 }}"
+    with new_test_keyspace(cql, ks_defs) as test_keyspace:
+        # enable_create_table_with_compact_storage is still enabled in db/config
+        with new_test_table(cql, test_keyspace, schema="p int PRIMARY KEY, v int", extra="WITH COMPACT STORAGE") as test_table:
+            pass
+
+@pytest.mark.parametrize("config_value", [False, True])
```

Comment:
nitpick: instead of calling this parameter "config_value", a name like "enable_compact_storage" would be much more self-explanatory.

Reply:
Will be fixed in v3